tests/unit/command/test_diff.py

```diff
@@ -162,10 +171,10 @@ def test_show_md():
         "| Status   | Path     |\n"
         "|----------|----------|\n"
         "| added    | file     |\n"
+        "| deleted  | zoo      |\n"
         "| deleted  | data{sep}    |\n"
-        "| deleted  | data{sep}bar |\n"
         "| deleted  | data{sep}foo |\n"
-        "| deleted  | zoo      |\n"
+        "| deleted  | data{sep}bar |\n"
```

Comment:
Hm, are you sure the order is correct? Maybe I'm missing something obvious.

Reply:
I simply edited tests after fixing logic 
Thanks for noticing @efiop . Will check.

Reply:
@efiop I remember now. Since we have already sorted entries in main method CmdDiff.run(), I don't want format specific methods to sort the entries. They take care only of the format to display, without altering any order. I kept the test with unordered entries, explicitly for this and Obviously I forgot 🤦 😄

Main method takes care of sorting, removing hash if not required, etc. Let me know if you want to make it clear. I can think of adding doc string, if required.

Thanks

Reply:
Ah, makes sense! Ok, let's leave it as is. Thank you!